### PR TITLE
SAC update: Stable-Baselines Version

### LINKED ISSUE #31: Policy loss in sac
In sac.py line 405 , Can you please explain the policy loss term
 loss_policy = (log_pi * (log_pi - q_value_new_actions + value_pred).detach()).mean() 

According to the SAC paper equation 12 it seems to me , it should be 
loss_policy = (log_pi - q_value_new_actions).mean()

### PR DESCRIPTION
**Pull request by araffin**
Fixes #31 
Add Stable-Baselines version of SAC
To be merged when stable-baselines 2.4.0 is out.

### CODE REVIEW

**Comment by araffin** [comment]
NOTE: the docker images should be updated...
